templates/lesson-template.md

```diff
@@ -0,0 +1,41 @@
+### Introduction
+
+A BRIEF INTRODUCTION.
+
+### Learning Outcomes
```

Comment:
In the lesson layout in `TOPStyleGuide.md`, this heading is called "Learning Overview". While here and in the `lesson-example.md`, this heading is called "Learning Outcomes". 

Is this intentional? Which heading should be used in the lesson?

Reply:
Good catch, thank you! Lesson Overview is the new verbiage we're going for.